api/server/server.go

```diff
@@ -17,18 +17,21 @@ import (
 	"github.com/docker/docker/api"
 	"github.com/docker/docker/autogen/dockerversion"
 	"github.com/docker/docker/daemon"
+	"github.com/docker/docker/pkg/authorization"
 	"github.com/docker/docker/pkg/sockets"
 	"github.com/docker/docker/pkg/version"
+	"net/http/httptest"
 )
 
 // Config provides the configuration for the API server
 type Config struct {
-	Logging     bool
-	EnableCors  bool
-	CorsHeaders string
-	Version     string
-	SocketGroup string
-	TLSConfig   *tls.Config
+	Logging      bool
+	EnableCors   bool
+	CorsHeaders  string
+	Version      string
+	SocketGroup  string
+	TLSConfig    *tls.Config
+	AuthZPlugins []string
```

Comment:
Thanks. I've replaced it in the go code except from daemon config.
I find it more clear to use ---authz-plugins=xxx than --authz-plugin-names=xxx, what do you think?